scrapyd_k8s/k8s_scheduler.py

```diff
@@ -0,0 +1,96 @@
+import logging
```

Comment:
Can we fit this in the directory structure? I wouldn't expect this in the src root.

Reply:
For me, this functionality seems related to the kubernetes launcher.

Reply:
k8s_scheduler is not the part of launcher because it is 1) a subscriber and should have this functionality to subscribe to the observer, 2) be the part which is initialized optionally when a user wants to limit the number of parallel job (launcher is not optional), 3) contains higher-level logic that uses low-level methods and helper methods from the launcher.

If you don't like that this file is located in the root, I can relocate it to the directory that belongs to this feature, say, limit_jobs or something like this, you can pick any name you like and I will add it.